doc/source/glossary.rst

```diff
@@ -226,23 +226,21 @@ Glossary
 
 
    broadcast
-       NumPy can do operations on arrays whose shapes are mismatched::
+       ``broadcasting`` is NumPy's ability to process ndarrays of
+       different sizes as if all were the same size.
 
-         >>> x = np.array([1, 2])
-         >>> y = np.array([[3], [4]])
+       When NumPy operates on two arrays, it works element by
+       element -- for instance, ``c = a * b`` is ::
 
-         >>> x
-         array([1, 2])
+           c[0,0,0] = a[0,0,0] * b[0,0,0]
+           c[0,0,1] = a[0,0,1] * b[0,0,1]
+          ...
 
-         >>> y
-         array([[3],
-                [4]])
-
-         >>> x + y
-         array([[4, 5],
-                [5, 6]])
+       Ordinarily this means the shapes of a and b must be identical. But in
```

Comment:
```suggestion
       Ordinarily, this means the shapes of a and b must be identical. But in
```

Reply:
[Sentence adverbs](https://www.merriam-webster.com/dictionary/sentence%20adverb) usually need the comma:
> Frankly, that's impossible.

but in this case it's stylistic:

> Ordinarily X is rational; on weekends it's irrational.
> Ordinarily, X is rational; on weekends, it's irrational.

and we can imagine a single text having both:

> We show that usually X is rational but on weekends it's irrational.
> Usually, X is rational, but, on weekends, it is irrational. QED.

Reply:
Thanks for giving me the name of this construction.

Both https://www.grammarly.com/blog/commas-after-introductory-words/ and https://theeditorsblog.net/2016/02/21/a-tale-of-adverbs-and-the-comma/ suggest that the comma should be present, so I'd err towards matching the norm rather than flouting stylistic freedom.

Reply:
I wouldn't define "norms" based on what's on the internet.

Reply:
The [guidance given by the Grammarly program (pg88)](http://onlinewritingcenters.org/wp-content/uploads/2019/01/2017-Dembsey-WCJ-36.1.pdf) appears to be:
> Introductory clauses and long phrases must always be followed by a comma, while single words and short phrases may or may not.

The second reference doesn't seem to say anything about this case.

Neither one seems to be supporting the comma campaign.